packages/x-components/src/x-modules/search/components/redirection.vue

```diff
@@ -1,17 +1,16 @@
 <template>
-  <div v-if="redirection && $scopedSlots.default" class="x-redirection" data-test="redirection">
+  <div v-if="redirection && slots.default" class="x-redirection" data-test="redirection">
```

Comment:
Not sure if this `slots.default` would work as you expect here. As slots management has changed and `slots.defaut `is now a function.